frontend/dispatcher.lua

```diff
@@ -70,7 +70,8 @@ local settingsList = {
     toggle_gsensor = {category="none", event="ToggleGSensor", title=_("Toggle accelerometer"), device=true, condition=Device:canToggleGSensor()},
     wifi_on = {category="none", event="InfoWifiOn", title=_("Turn on Wi-Fi"), device=true, condition=Device:hasWifiToggle()},
     wifi_off = {category="none", event="InfoWifiOff", title=_("Turn off Wi-Fi"), device=true, condition=Device:hasWifiToggle()},
-    toggle_wifi = {category="none", event="ToggleWifi", title=_("Toggle Wi-Fi"), device=true, condition=Device:hasWifiToggle(), separator=true},
```

Comment:
Should these `Wi-Fi`s be changed, too?

Reply:
There are ca. 30 strings containing `Wi-Fi` in frontend and plugins. Shall I replace all with `wifi`. I'm asking, because of the translation effort, that will following this change.

Reply:
I'm firmly in camp change all Wi-Fi to wifi. But that's neither here nor there wrt this PR, of course. ;-P

Reply:
But... but... what will the Wi-Fi Alliance say?! :D.

Snark aside, it's not a hill i'm willing to die on, but I firmly prefer Wi-Fi ;). Then again, I'm that weird guy that likes CamelCase labels everywhere, so ;p.